crowdin_api/api_resources/users/resource.py

```diff
@@ -58,6 +58,32 @@ class UsersResource(BaseUsersResource):
     https://developer.crowdin.com/api/v2/#tag/Users
     """
 
+    def list_project_members(
+        self,
+        projectId: Optional[int] = None,
+        search: Optional[str] = None,
+        role: Optional[UserRole] = None,
+        languageId: Optional[str] = None,
+        page: Optional[int] = None,
+        offset: Optional[int] = None,
+        limit: Optional[int] = None,
+    ):
+        """
+        List Project Members.
+
+        Link to documentation:
+        https://developer.crowdin.com/api/v2/#operation/api.projects.members.getMany
+        """
+        return self._list_project_members(
+            projectId=projectId,
+            search=search,
+            languageId=languageId,
+            page=page,
+            offset=offset,
+            limit=limit,
+            extraParams={"role": role}
```

Comment:
What if the role is not passed? It will still add it to the extraParams?

Reply:
Same as before if the role is not passed, it's added to params with `None` value/ 
line 40 before
```
params = {"search": search, "role": role, "languageId": languageId}
```

Also, this is added in the first test scenario in test_list_project_members for the UserResource.